src/scss/_responsive-flat.scss

```diff
@@ -0,0 +1,64 @@
+////
+/// @group o-table
+/// @link http://registry.origami.ft.com/components/o-table
+////
+
+/// Make a table 'responsive-flat'
+@mixin oTableResponsive-flat {
+	*,
+	*:before, 
+	*:after {
+		box-sizing: border-box;
+	}
+
+	display: block;
```

Comment:
Think this should be at the start of the mixin as well as the media query block, then followed by the `*` block...